File: db_connection.py
import mysql.connector
from mysql.connector import Error

def get_db_connection():
    try:
        conn = mysql.connector.connect(
            host='localhost',
            user='root',
            password='',
            database='steam'
        )
        if conn.is_connected():
            return conn
    except Error as e:
        print(f"Error: {e}")
    return None
    import mysql.connector
from mysql.connector import Error

def get_db_connection():
    try:
        conn = mysql.connector.connect(
            host='localhost',
            user='root',
            password='',
            database='steam'
        )
        if conn.is_connected():
            return conn
    except Error as e:
        print(f"Error: {e}")
    return None

File: www/functions/db_connection.php
<?php
/**
 * Author: Hilal Cubukcu
 * Last modified on: 12.05.2024
 * Title: Database Connection
 * Summary: This script establishes a connection to the MySQL database.
 */

$servername = "localhost";
$name = "root"; 
$password = ""; 
$database = "steam"; 

//Create connection
$conn = mysqli_connect($servername, $name, $password, $database);

// Check connection
if (!$conn) {
    echo "Connection failed!";
}
?>
